compiler/rename/RnExpr.hs

```diff
@@ -2198,7 +2198,7 @@ getMonadFailOp
 -- rec. Hence, compared to the ghc implementation, the arguments are switched here.
 mkSetField :: Name -> LHsExpr GhcRn -> Located FieldLabelString -> LHsExpr GhcRn -> HsExpr GhcRn
 mkSetField set_field a (L _ field) b =
-  genHsApp (genHsApp (genHsVar set_field `genAppType` genHsTyLit field)  b) a
+  genHsPar (genHsApp (genHsApp (genHsVar set_field `genAppType` genHsTyLit field) b) a)
```

Comment:
I'm not sure how to test this one, since this takes place after parsing. In other words, the result of `damlc desugar` has a plain record update, `create this {owner = newOwner}`.